yeelight/flow.py

```diff
@@ -67,7 +71,7 @@ def as_list(self):
         :rtype: list
         """
         # Duration must be at least 50, otherwise there's an error.
-        return [max(50, self.duration), self._mode, self._value, self.brightness]
+        return [max(50, self.duration), self._mode, self._value, int(self.brightness)]
```

Comment:
Should we not do this when writing to brightness, to avoid having to pepper `int()` calls everywhere?